packages/block-editor/src/components/block-actions/index.js

```diff
@@ -51,7 +51,8 @@ export default compose( [
 			return (
 				!! block &&
 				hasBlockSupport( block.name, 'multiple', true ) &&
-				canInsertBlockType( block.name, rootClientId )
+				canInsertBlockType( block.name, rootClientId ) &&
+				hasBlockSupport( block.name, 'duplicate', true )
```

Comment:
I think `duplicable`, although kind of a mouthful, would probably convey better the meaning of this property.
(Hopefully there's a better English word though 😅)